src/LogExpert.Tests/DateFormatParserTest.cs

```diff
@@ -15,15 +18,24 @@ public void CanParseAllCultures()
         {
             var cultures = CultureInfo.GetCultures(CultureTypes.AllCultures);
 
+           // HashSet<string> exclude = ["dz", "ckb-IR", "ar-SA", "lrc" , "lrc-IR", "mzn" , "mzn-IR", "ps"];
+
             foreach (var culture in cultures)
             {
-                if (culture.Name == "dz" || culture.Name.StartsWith("dz-"))
+                if (culture.Name == "dz" || culture.Name == "ar" ||  culture.Name.StartsWith("ar-") || culture.Name.StartsWith("dz-"))
```

Comment:
ar is arabic? 
please adapt the console message, so this is reflected as well

Reply:
Done